starter-code/public/scripts/controllers/articleController.js

```diff
@@ -3,10 +3,16 @@ var app = app || {};
 
 (function(module) {
   const articleController = {};
-  // TODO: Setup a function that kicks off the fetching and rendering of articles, using the same
+  // DONE: Setup a function that kicks off the fetching and rendering of articles, using the same
   // code that used to be in index.html.
-  // Also be sure to hide all the main section elements, and reveal the #articles section:
+  // Also be sure to hide all the main section elements, and reveal the #articles section
+  // estimated time 10 minutes | actual 12 minutes
 
+  articleController.initArticleView = function (){
+    $('.tab-content').hide();
+    $('#articles').show();
+  }
+  app.Article.fetchAll();
```

Comment:
This call needs to be inside of your initArticleView method. Also .fetchAll needs a callback function. What do we call to render our article data to the page? Hint Hint: We use to call it inside of our index page.